scipy/stats/tests/test_qmc.py

```diff
@@ -923,6 +881,18 @@ def test_NormalQMCSeeded(self):
                                      [-0.997875, 0.399134, 1.032234]])
         assert_allclose(samples, samples_expected, atol=1e-4)
 
+        # same test with another engine
```

Comment:
Is it really another engine, or is an equivalent engine that happens to be passed in directly rather than it being created internaly?

Reply:
It's the same engine so that we have the same output. I used the same wording as with the other tests for now.

I tried to not do much on the test and just did the minimum to reflect the changes, but this is definitely not in a good state and should be reworked.